generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/_auth-expired.interceptor.ts

```diff
@@ -36,81 +36,86 @@ import { LoginModalService } from '../../shared/login/login-modal.service';
 import { StateStorageService } from '../../shared/auth/state-storage.service';
 <%_ } _%>
 
-export class AuthExpiredInterceptor extends JhiHttpInterceptor {
+export class AuthExpiredInterceptor implements HttpInterceptor {
 
-<%_ if (authenticationType === 'jwt' || authenticationType === 'uaa') { _%>
+<%_ if (authenticationType === 'jwt') { _%>
+    constructor(private injector: Injector) {
+    }
+<%_ } else if (authenticationType === 'uaa') { _%>
```

Comment:
why this branching both constructor look the same so just do `['jwt', 'uaa'].includes(authenticationType)` check